sample/002-microservice-app/src/health/health.controller.ts

```diff
@@ -26,7 +27,10 @@ export class HealthController {
         this.microservice.pingCheck<RedisOptions>('redis', {
           transport: Transport.REDIS,
           options: {
-            url: 'redis://localhost:6379',
+            ...parseURL('redis://localhost:6379'),
+            // or use the ioreids options directly
+            // host: 'localhost',
+            // port: 16379,
```

Comment:
DONE.

Reply:
Since [nestjs IORedisOptions](https://github.com/nestjs/nest/blob/009f6d2fc3881aa23a3b247c33ee3dcebea1f50f/packages/microservices/external/redis.interface.ts#L6) doesn't seem to contain host and port, I removed it. but the [ioredis IORedisOptions](https://github.com/luin/ioredis/blob/07ee6ead7eda2d8d60b260b227dd8d29322eb81c/lib/redis/RedisOptions.ts#L184)  contain them.